backend/src/StamAcasa.IdentityServer/Quickstart/Account/DeleteAccountController.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using IdentityServer.Data;
+using IdentityServer4;
+using IdentityServer4.Models;
+using IdentityServer4.Services;
+using IdentityServer4.Validation;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+
+namespace StamAcasa.IdentityServer.Quickstart.Account
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [AllowAnonymous]
+    public class DeleteAccountController : ControllerBase
+    {
+        private readonly UserManager<ApplicationUser> _userManager;
+        private readonly DefaultTokenService _tokenService;
+        private readonly IStamAcasaIdentityConfiguration _identityConfiguration;
+        private readonly IHttpClientFactory _clientFactory;
+        private readonly string _apiUrl;
+
+        public DeleteAccountController(UserManager<ApplicationUser> userManager, DefaultTokenService tokenService, IStamAcasaIdentityConfiguration identityConfiguration, IHttpClientFactory clientFactory, IConfiguration configuration)
+        {
+            _userManager = userManager;
+            _tokenService = tokenService;
+            _identityConfiguration = identityConfiguration;
+            _clientFactory = clientFactory;
+            _apiUrl = configuration["StamAcasaApi"];
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> DeleteAccountAsync([FromBody] DeleteAccountModel model)
+        {
+            var user = await _userManager.FindByNameAsync(model.Username);
```

Comment:
we should not provide info to anyone if a user with specified username exists or not

Reply:
Yes I agree with this one.
Can you please make the change @irinel-nistor ?